lua/rustowl/highlight.lua

```diff
@@ -2,12 +2,13 @@ local M = {}
 
 local hl_ns = vim.api.nvim_create_namespace('rustowl')
 
----@param bufnr? number
+---@param bufnr number
+---@param line number
+---@param col number
 function M.enable(bufnr, line, col)
```

Comment:
```suggestion
---@param line number
---@param col number
---@param bufnr? number
function M.enable(line, col, bufnr)
```

The `bufnr` is optional since we fall back to the current buffer if it's `nil`. Moving it to the last column would allow a caller to not pass it in.